studio/tsconfig.json

```diff
@@ -0,0 +1,30 @@
+{
+  "compilerOptions": {
+    "target": "es5",
+    "lib": [
+      "dom",
+      "dom.iterable",
+      "esnext"
+    ],
+    "allowJs": true,
+    "skipLibCheck": true,
+    "strict": true,
+    "forceConsistentCasingInFileNames": true,
+    "noEmit": true,
+    "esModuleInterop": true,
+    "module": "esnext",
+    "moduleResolution": "node",
+    "resolveJsonModule": true,
+    "isolatedModules": true,
+    "jsx": "preserve",
+    "baseUrl": "."
+  },
+  "include": [
```

Comment:
Thoughts on using TS paths to reduce the `../../components` imports?

```json
"paths": {
    "@components": ["components"],
}
```

Reply:
Next.js actually supports this out-of-the-box with tsconfig: https://nextjs.org/docs/advanced-features/module-path-aliases

we won't need to add `@components` - we can just write `components/XXX` and it should work 👍

Reply:
Roger! Yeah, the baseUrl, in this case, would solve that!